webpack.config.js

```diff
@@ -1,31 +1,36 @@
-var webpack = require("webpack");
-var UglifyJsPlugin = require("uglifyjs-webpack-plugin");
-var path = require("path");
+const webpack = require("webpack");
+const ExtractTextPlugin = require('extract-text-webpack-plugin')
+const UglifyJsPlugin = require("uglifyjs-webpack-plugin");
+const path = require("path");
 
-var srcDir = path.resolve(__dirname, "src");
-var scriptsDir = srcDir + "/scripts";
-var assetsDir = srcDir + "/assets";
-var stylesDir = srcDir + "/styles";
-var buildDir = path.resolve(__dirname, "build");
-var testDir = path.resolve(__dirname, "test");
-var unitTestsDir = testDir + "/unit-tests";
-var functionalTestsDir = testDir + "/functional-tests";
+const srcDir = path.resolve(__dirname, "src");
+const scriptsDir = srcDir + "/scripts";
+const embedDir = scriptsDir + "/embed";
+const assetsDir = srcDir + "/assets";
+const stylesDir = srcDir + "/styles";
+const buildDir = path.resolve(__dirname, "build");
+const testDir = path.resolve(__dirname, "test");
+const unitTestsDir = testDir + "/unit-tests";
+const functionalTestsDir = testDir + "/functional-tests";
 
-var prod = process.env.NODE_ENV === "production";
+const prod = process.env.NODE_ENV === "production";
 
 
-var config = {
-  entry: prod
-    ? [scriptsDir + "/index.js"]
-    : [
-        "react-hot-loader/patch",
-        scriptsDir + "/index.js",
-        "webpack-hot-middleware/client?quiet=true"
-      ],
+const config = {
+  entry: {
+    bundle: prod
+      ? [scriptsDir + "/index.js"]
+      : [
+          "react-hot-loader/patch",
+          scriptsDir + "/index.js",
+          "webpack-hot-middleware/client?quiet=true"
+        ],
+    'embed/bundle': embedDir + "/client/index.js"
```

Comment:
This is how we can specify multiple entry points that will be bundled into separate outputs